examples/oneapi/cpp/source/decision_forest/df_cls_dense_batch.cpp

```diff
@@ -0,0 +1,102 @@
+/*******************************************************************************
```

Comment:
It is not very consistent with another classifier - SVM, KNN. Let's either align according to your example or SVM or KNN. @agorshk see example for DF

Reply:
fixed